.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: NodeJS with Grunt
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Build
+      run: |
+        npm install
```

We have a `yarn.lock` file so the dependencies should be installed with Yarn.
```suggestion
        yarn install
```